inductiva/molecules/scenarios/protein_solvation/protein_solvation.py

```diff
@@ -85,7 +87,8 @@ def simulate(
 
         replace_params_in_template(self.template_dir, "commands.json.jinja", {
             "pdb_file": self.protein_pdb,
-            "charged": self.charged
+            "charged": self.charged,
+            "selected_part": visualized_section
```

Comment:
Why are you giving these parameters/variables different names, if you can control both?